packages/es-components/src/components/navigation/horizontalnav/HorizontalNavItem.md

```diff
@@ -0,0 +1 @@
+The `HorizontalNavItem` component is used within the HorizontalNav component.
```

Comment:
I lied; one more question. We do not provide documentation for `Tab` as it is part of `TabPanel.Tab`. Is there a need to provide the documentation for `HorizontalNavItem`?

Reply:
Well, we do it with DrawerPanel, so there's some precedent. We don't do it with Modal though so we've been inconsistent.